app/src/main/java/org/fossasia/openevent/general/sessions/sessiontype/SessionType.kt

```diff
@@ -1,17 +1,27 @@
-package org.fossasia.openevent.general.sessions
+package org.fossasia.openevent.general.sessions.sessiontype
 
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 import com.fasterxml.jackson.databind.PropertyNamingStrategy
 import com.fasterxml.jackson.databind.annotation.JsonNaming
 import com.github.jasminb.jsonapi.LongIdHandler
 import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Relationship
 import com.github.jasminb.jsonapi.annotations.Type
+import org.fossasia.openevent.general.sessions.Session
 
 @Type("session-type")
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
+@Entity
 data class SessionType(
     @Id(LongIdHandler::class)
+    @PrimaryKey
     val id: Long,
     val name: String,
     val length: String?,
-    val deletedAt: String?
+    val deletedAt: String?,
+    @ColumnInfo
+    @Relationship("session")
+    var session: Session? = null
```

Comment:
Remove this relationship also as this relationship will never be used and will create a circular dependency nothing else